### PR TITLE
Prepare Release 3.3.0

### PR DESCRIPTION
**Pull request by iangmaia**
This version contains:
- Update to make `publish_private_pod` support `--allow-warnings`: #86
- Fixing Ruby, RuboCop and RSpec versions on CI (added on this PR, #88)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,24 @@ _None._
 
 _None._
 
+## 3.3.0
+
+### Breaking Changes
+
+_None._
+
```
Comment (AliSoftware) [inline CHANGELOG.md]: We usually remove empty sections when releasing the version
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,6 +48,24 @@ _None._
...
+### New Features
+
+- Make `publish_private_pod` support `--allow-warnings`. [#86]
+
+### Bug Fixes
+
+_None._
+
```
Comment (AliSoftware) [inline CHANGELOG.md]: Ditto
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,6 +48,24 @@ _None._
...
+### Internal Changes
+
+- Fixed Ruby, RuboCop and RSpec versions on CI
```
Comment (AliSoftware) [inline CHANGELOG.md]: At first I wondered where that entry came from as I didn't remember seeing a PR for it recently. Then I realized you did this as part of this very PR 😛

```suggestion
- Fixed Ruby, RuboCop and RSpec versions on CI [#88]
```

**Review by AliSoftware — APPROVED** [review]
Approved but disabled auto merge to let you the opportunity to fix suggestions I left in CHANGELOG.

File: CHANGELOG.md (hunk 1 above)
Comment (iangmaia) [reply CHANGELOG.md]: Thanks! Updated on 38dc91d68212eb6eeceef7a49ffee06c8191d758.

File: CHANGELOG.md (hunk 3 above)
Comment (iangmaia) [reply CHANGELOG.md]: Ah yeah, CI was breaking on this PR and I noticed it was due to using the latest version of RuboCop (potentially a bug, as I've seen it reported for older versions). Updated with this change on 38dc91d68212eb6eeceef7a49ffee06c8191d758.

**Pull request merged**